lib/parser.js

```diff
@@ -3,7 +3,7 @@
 var parse = require('espree').parse;
 var doctrine = require('doctrine');
 var _ = require('underscore');
-var traverse = require('estraverse').traverse;
+var traverse = require('estraverse-fb').traverse;
 var Syntax = require('estraverse').Syntax;
```

Comment:
Use `require('estraverse-fb').Syntax` to ensure it's JSX-injected Syntax.
Fix `lib/visitor.js` too.